.github/workflows/test.yml

```
@@ -0,0 +1,26 @@
+name: Run Steampipe awstf compliance
+on: [pull_request,push]
+
+jobs:
+  steampipe-awstf-compliance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Setup steampipe
+        id: setup-steampipe
+        uses: turbot/steampipe-action-setup@v1.4.0
```

```suggestion
        uses: turbot/steampipe-action-setup@v1
```

Can we point to just v1 instead of a specific version?

---

This would require us to maintain and update the existing `v1` [tag](https://github.com/turbot/steampipe-action-setup/releases/tag/v1) as referencing this tag gives us an outdated version of the action at the moment which doesn't have `plugin-connections` input.

---

Whoops forgot to give example, you can see [here](https://github.com/aws-actions/configure-aws-credentials/releases) they publish the `v3` or `v4` tag in-step with an actual specific version tag also, essentially just moving it along to the latest commit in that majors series.

---

Discussed: we will update the major tags so update to `v1`

---

Updated versions in the `readme` haven't done in the test workflow yet as would cause failure until tag is updated on setup repo.